src/Rules/Classes/EnumSanityRule.php

```diff
@@ -69,6 +70,15 @@ public function processNode(Node $node, Scope $scope): array
 				}
 			}
 
+			// serialize/unserialize is not considered a magic method by PhpParser
```

Comment:
php-parser adds these method in the next release with https://github.com/nikic/PHP-Parser/pull/860